.github/workflows/main.yml

```
@@ -149,8 +149,9 @@ jobs:
       run: |
         pandoc -s -o README.md README.rst
     - name: PyPI upload
-      uses: pypa/gh-action-pypi-publish@v1.9.0
+      uses: pypa/gh-action-pypi-publish@v1.10.0
```

v1.10.0 has a bug with attestations. This release has the fix:
```suggestion
      uses: pypa/gh-action-pypi-publish@v1.10.1
```